webapp/src/components/root/root.jsx

```diff
@@ -49,9 +54,12 @@ export default class Root extends React.Component {
     }
 
     submit = () => {
-        const {submit, close, postID} = this.props;
+        const {submit, update, close, postID, selectedPost} = this.props;
         const {message, sendTo, attachToThread} = this.state;
-        if (attachToThread) {
+        const isEditing = selectedPost !== null;
+        if(isEditing){
+            update(message, sendTo, postID);
```

Comment:
I feel that when editing a todo, we should not allow to send or attach it to a thread. Nevertheless, I would like to hear the thoughts of @aaronrothschild regarding this.

Reply:
> I feel that when editing a todo, we should not allow to send or attach it to a thread. Nevertheless, I would like to hear the thoughts of @aaronrothschild regarding this.

I think we shouldn't send an update automatically.  If the user optionally, provides a message explaining the change then an update should be pushed by default.  

I'm 0/5 on this - as Dani has noted before the user can also easily add a note to the thread as well.